cypress/support/commands.js

```diff
@@ -23,3 +23,13 @@
 //
 // -- This will overwrite an existing command --
 // Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
+
+Cypress.Commands.add('login', () => {
+  const username = 'Snow228';
+  const password = 'Snow228!';
+
+  cy.get('#userName').should('exist').type(username);
```

Comment:
```suggestion
  cy.get('#userName').type(username);
```
no need to check the existence, if there is no such field cypress won't be able to type in it